docs/ref/settings.txt

```diff
@@ -3666,6 +3668,9 @@ Forms
 * :setting:`FORM_RENDERER`
 * :setting:`FORMS_URLFIELD_ASSUME_HTTPS`
 
+
+.. _globalization-settings:
```

Comment:
This is a new unused reference, I believe the start of implementing [this comment](https://github.com/django/django/pull/18028#discussion_r1567427192).